### ISSUE TITLE
Some questions about the experimental.

### ISSUE THREAD

**Issue by FelixHuiweiLin** [mmahesh/variants_of_rmsprop_and_adagrad#3]
1.How to calculate the global optimum  p ∗ in the  relative suboptimality plots and  regret plots?
2.During your model training, do you use training data mini-batch or one by one？

**Comment by mmahesh** [comment]
1) You can run some known algorithm such as Newton method or LBFGS or Gradient Descent for a long time then obtain p*, if the problem is convex. You can then use that p* in relative suboptimality plots or regret plots.   For nonconvex problems, as a heuristic you can use the least function value attained by any of the algorithms (if you use sufficiently large number of iterations).
2) We used mini-batching in general, however, for regret plots we use one by one.

**Comment by FelixHuiweiLin** [comment]
Thanks!
